### ISSUE TITLE
A11y_Azure Data Studio_Toolbar_Top Operations_ScreenReader: NVDA is not announcing column header association for any of the data cell. 

### ISSUE THREAD

**Issue by Rama8345** [microsoft/azuredatastudio#20780]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #SQL Azure Data Studio; #SH-SQL Azure Data Studio-Win32-OCT2022; #DesktopApp; #ScreenReader; #WCAG1.3.1; #FTP; #NVDA

### Environment Details:
Application Name: Azure Data Studio
Windows Version: Windows 11

### Repro Steps:
1. Launch the "Azure data studio" application.
2. Connect with the provided SQL server.
3. TAB to "New Query" button and hit ENTER key.
4. Enter the SQL query and TAB to "Estimated Plan" button and hit ENTER key.
5. "Query Plan (Preview) screen appear.
6. Tab to "Top operations" control from the right navigation and tab to any of the data cell.
7. Observe that NVDA is not announcing column header association for any of the data cell. 

### Actual:
NVDA is not announcing column header association for any of the data cell. 

**NOTE: Similar issue is observed throughout the application.**

### Expected Result:
NVDA should announce column header association for every data cell. 

### Attachment:
![A11y_Azure Data Studio_Toolbar_Top Operations_ScreenReader](https://user-images.githubusercontent.com/115216656/194817455-61e4e665-7a61-4995-8993-c7baa983a40d.gif)

**Comment by aasimkhan30** [comment]
@Rama8345, this issue is fixed in the latest insiders. Please verify it

**Comment by Rama8345** [comment]
Verified the bug in the latest insider build. The issue is fixed. So, closing the bug. Please find below attachment for reference.
![20780_Closed](https://user-images.githubusercontent.com/115216656/204990525-4f5f6c15-8fb6-4221-8757-2f64a4b69b07.gif)

**Comment by 12-shweta** [comment]
Github Tags:#Closed;
